integration-tests/common-test-code/src/main/java/com/vaadin/flow/quarkus/it/routecontext/DetailApartView.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2000-2018 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.vaadin.flow.quarkus.it.routecontext;
+
+import javax.annotation.PostConstruct;
+import javax.inject.Inject;
+
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.Label;
+import com.vaadin.flow.router.AfterNavigationEvent;
+import com.vaadin.flow.router.AfterNavigationObserver;
+import com.vaadin.flow.router.Route;
+import com.vaadin.flow.router.RouterLink;
+import com.vaadin.quarkus.annotation.RouteScopeOwner;
+
+@Route(value = "apart", layout = MasterView.class)
+public class DetailApartView extends AbstractCountedView
+        implements AfterNavigationObserver {
+
+    public static final String MASTER = "master";
+    public static final String BEAN_LABEL = "BEAN_LABEL";
+
+    @Inject
+    @RouteScopeOwner(DetailApartView.class)
+    private ApartBean apartBean;
```

Comment:
For this 'private' modifier my IDE gives a warning:
`Private access modifier usage is not recommended for Quarkus CDI members`.
```suggestion
   ApartBean apartBean;
```

Reply:
done